### ISSUE TITLE
Propose short list of 5 initiatives to reach out 

### ISSUE THREAD

**Issue by ramlyed** [therabble/sustain-care.org#50]

**Comment by ramlyed** [comment]
@wrolson22 will help with ideas

**Comment by mfeif** [comment]
So this might be obvious to all but I'd like to propose some guidelines to how we choose. Just my opinion here, (represent no particular hierarchy though they are ordered) and I am open to what I may have overlooked:

1. Projects that have some sufficient scale/momentum.
2. that embody values we have. For FoodWorks, this needs to include social equity in some measure to be in accordance with our tax-exempt status. (shouldn't be a problem)
   a. In particular, _ideally_ this means orienting our attentions towards people of color and other marginalized groups in and around Madison 
3. that practice good safety protocols (as far as we can tell)
4. that service a genuine need (some notion of efficiency)
5. that serve one or more of these populations:
   a. workers who are in crisis mode work w/r/t COVID-19 (medical, core infrastructure, core services)
   b. people who have been made food insecure because of COVID-19
   c. people who already experienced food insecurity
6. that help keep a local business open (preferably small business)
7. that pay decently well, and are using the project to keep people employed
8. that source from local producers and growers and small-business distribution networks
9. that are cooperating well with their peers and delivery and hospitals and so on: citizenship award sort of thing
10. that represent a project that can grow into a more semi-permanent service that will endure as long as necessary (as opposed to one-off deliveries)

**That's a long list. Nobody will get ALL of them. The best choices will tick several.** We might also aim initially for some *variety* to keep our platform value proposition attractive to the widest possible audience... to help get the next wave(s) of people to sign up.

**Comment by wrolson22** [comment]
This is a great list. I agree, like you said, there is no way to tick all of these for every case, but are there three main checks that they absolutely have to tick in order for us to consider further? This may be a good way of self vetting so we don't get a bunch of random stuff we have to sift through. 

That is likely just a problem down the road but may be good to get into place up front. I think the top three for me would be: 2, 5 and 7 (although I think 10 is also important).

**Comment by jchuran** [comment]
I would propose:
1. REAP Food Group
2. Little John's - Dave H's project 

It would be nice to get to our goal of helping more small independent places.  The Morris Ramen initiative come to mind.  It would be great to get some places that are out of our network.

**Comment by mfeif** [comment]
@wrolson22 Good prioritization; I agree. I've re-ordered the list as-such. I don't think we'll have too difficult a time getting started, though we indeed might get some not-so-serious entries down the road. I expect we're mostly in alignment with one another here.

@jchuran I think those two are on my informal list of favorites too, so let's maybe consider them  the beginning of our first winners.

And out of network is a good point. I heard that Metcalfe's is doing "something." The article that Will shared a while back has a bunch of things on it I hadn't heard about.

I do think a "smaller" one like what we've heard about Morris would be a good sample starter too. Good thinking.

**Comment by jchuran** [comment]
@mfeif Sorry, I may be confused.  Are we coming up with a list of 5 initiatives or a list of 5 criteria an initiative must meet?

**Comment by mfeif** [comment]
Sorry @jchuran yes, I've been unclear.

We are coming up with 5 "projects/initiatives". I just wanted to put in here some notion of how we choose. I'm **super comfy** starting the list here:

1. REAP Food Group: [Farms to Families](http://reapfoodgroup.org/farms-to-families/)
1. Little John's: [Need More Info](https://littlejohnsrestaurant.com/donate)
1. Morris Ramen / Goodman Center
1. ?
1. ?

Any projects standing out to you, @wrolson22 ?

**Comment by wrolson22** [comment]
Feed a hero, help save a restaurant. Launched by the Madison Area Chef's Network. Looks like they are doing well on go fund me but they will run out of funds eventually. The others look good too! 

https://www.gofundme.com/f/help-a-hero-save-a-restaurant?utm_source=facebook&utm_medium=social&utm_campaign=p_cp%20share-sheet&fbclid=IwAR27j2qwhXkvTNf1vWnzndtBpgaZgVfP2gt0nyOH26m-Xhv2ad3Xg0vGbVo

**Comment by mfeif** [comment]
I'm trying to learn more about the Feed a Hero project; I reached out to Chef Miller, but haven't heard back yet. I'm looking for some detail about the service itself rather than just the funding.

I talked to Morris Ramen again; they are ideal, and are not delivering also to the Bayview Food Pantry, which serves a lot of Asian-American people and Asian immigrants. Morris is able to produce culturally appropriate meals, which is super exciting.

I'd like to add FEED's project to the short-list

1. REAP Food Group: [Farms to Families](http://reapfoodgroup.org/farms-to-families/)
2. Little John's: [Need More Info](https://littlejohnsrestaurant.com/donate)
3. Morris Ramen / Goodman Center
4. FEED to Go (https://feedkitchens.org/feedtogo/)
5. Urban Triage project that we've been mentioning.

We can always add more.

I've been sprinting on the site; MVP is almost done, so I think we can start reaching out in the next few days.
